### PR TITLE
chore: add FaceLivenessDetector canary

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- PR to enable the faceLivenessDetector canary now that things are available on npm

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by reesscot — DISMISSED** [review]

**Review by ioanabrooks — APPROVED** [review]

**Review by 0618 — APPROVED** [review]

File: canary/apps/react/next/package.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
   "size-limit": [
     {
       "path": ".next/static/**/*.js",
-      "limit": "350 kB"
+      "limit": "500 kB"
```
Comment (0618) [inline canary/apps/react/next/package.json:28]: Somehow the size became 521 😂 

https://github.com/aws-amplify/amplify-ui/actions/runs/4661483627/jobs/8250800510?pr=3693

Comment (0618) [inline canary/apps/react/next/package.json:28]:
```suggestion
      "limit": "530 kB"
```

**Review by 0618 — DISMISSED** [review]
PoC workflows passed https://github.com/aws-amplify/amplify-ui/pull/3693

**Review by reesscot — APPROVED** [review]

File: canary/apps/react/next/pages/index.js [hunk 2]
```diff
@@ -20,6 +22,11 @@ function App() {
         <Button variation="primary">Click me!</Button>
         <Button onClick={signOut}>Sign out</Button>
       </Card>
+      <FaceLivenessDetector
+        sessionId={'123'}
+        region={'us-east-1'}
```
Comment (wlee221) [inline canary/apps/react/next/pages/index.js:27]:
```suggestion
        sessionId="123"
        region="us-east-1"
```

File: canary/apps/react/next/package.json [hunk 3]
```diff
@@ -24,7 +25,7 @@
   "size-limit": [
     {
       "path": ".next/static/**/*.js",
-      "limit": "350 kB"
+      "limit": "530 kB"
```
Comment (wlee221) [inline canary/apps/react/next/package.json:28]: How was this number determined?

Comment (thaddmt) [reply canary/apps/react/next/package.json:28]: MJ ran the build on a PoC PR and saw that the value increased - https://github.com/aws-amplify/amplify-ui/pull/3693

@0618 do you have the link to the failing run where we saw the new value?

Comment (0618) [reply canary/apps/react/next/package.json:28]: Sure, that was this build

https://github.com/aws-amplify/amplify-ui/actions/runs/4661483627/jobs/8250800510

image

**Review by 0618 — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

**Pull request merged**